readme.md

```diff
@@ -29,10 +29,14 @@ $ imagemin --help
     $ imagemin foo.png > foo-optimized.png
     $ cat foo.png | imagemin > foo-optimized.png
     $ imagemin foo.png --plugin=pngquant > foo-optimized.png
-    $ imagemin foo.png --plugin.pngquant.quality={0.1,0.2} > foo-optimized.png
+    $ imagemin foo.png --plugin.pngquant.quality=0.5 --plugin.pngquant.quality=1 > foo-optimized.png
     $ imagemin foo.png --plugin.webp.quality=95 --plugin.webp.preset=icon > foo-icon.webp
 ```
 
+**macOS** users can also use the short CLI syntax for array arguments:
```

Comment:
I think `non-Windows` would be more correct than `macOS`

Reply:
Just a typo. It should be `macOS/linux`. But ok, `non-Windows` is also fine.